### PR TITLE
rpi-eeprom-update: minor fixes to usage, rename version check function

### PR DESCRIPTION
**Pull request by andrum99**

### CODE REVIEW

File: rpi-eeprom-update [hunk 1]
```diff
@@ -193,22 +193,22 @@ checkDependencies() {
    fi
 
    if vcgencmd bootloader_config | grep -qi "Command not registered"; then
-      die "vcgencmd: bootloader_config. not supported. Please update VC firmware"
+      die "vcgencmd: 'bootloader_config` command not supported. Please update VC firmware"
```
Comment (XECDesign) [inline rpi-eeprom-update]: Is it just me or do those single quotes not match? Shouldn't they just be normal `'` characters?

File: rpi-eeprom-update [hunk 2]
```diff
@@ -230,9 +230,9 @@ rpi-eeprom-update [options]... [FILE]
    A backup of the current EEPROM config file is written to ${FIRMWARE_BACKUP_DIR}
    before applying the update.
 
-   -a Install the latest critical update if necessary.
-   -d Use the default bootloader config instead of migrating the current settings.
-   -f Install the given file instead of the latest critical update.
+   -a Install the latest update if necessary
+   -d Use the default bootloader config instead of migrating the current settings
+   -f Install the given file instead of the latest critical update
```
Comment (MilhouseVH) [inline rpi-eeprom-update]:
```suggestion
   -f Install the given file instead of the latest ${FIRMWARE_RELEASE_STATUS } update
```

Comment (andrum99) [reply rpi-eeprom-update]: The usage message is used to build a man page when the package is installed, so this wouldn't work. I've changed 'critical' to 'applicable' instead.

**Comment by timg236** [comment]
Thanks, I've given this a quick test and the updates look good

**Pull request merged**